.github/workflows/pr-title-checker.yml

```diff
@@ -0,0 +1,17 @@
+name: "PR Title Checker"
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+
+jobs:
+  check:
+    name: Validate PR Title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: thehanimo/pr-title-checker@v1.4.3
```

Comment:
Thank you for the suggestion, but I never saw anyone do that, and I think that this may be an overkill.
I'll have to update all of my other workflows, to follow the same format.
Also, this workflow doesn't carry a lot of permissions, so the security risks are slim.

Reply:
Can you re-review, please.